docs/root/configuration/http/http_filters/stateful_session_filter.rst

```diff
@@ -0,0 +1,75 @@
+.. _config_http_filters_stateful_session:
+
+Stateful session
+================
+
+Stateful session is an HTTP filter which sets an override host based on the extensible session state
+and updates the session state based on the final selected upstream host. The override host will
+eventually overwrites the load balancing result. This filter can implement session sticky without using
+a hash-based load balancer.
+And by extending the session state, this filter also allows more flexible control over the results of
```

Comment:
Sorry, I think we can add a new line here.

Reply:
get it.